app/views/foods/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>New food</h1>
+
```

Comment:
- [ ] Kindly ensure that you follow the design templates for all the views as this is a requirement. This means following how components are displayed in all the given designs to match with your designs. [Link](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md)

Recepies
![image](https://user-images.githubusercontent.com/81932478/209352497-6bac2bb5-db1b-4304-a2a0-b8a72ce2f590.png)

![image](https://user-images.githubusercontent.com/81932478/209352532-20d9331b-c52b-4596-8653-a382135e7acb.png)

![image](https://user-images.githubusercontent.com/81932478/209352582-99abbf46-e4d7-4110-b9e9-811f54d766b1.png)

Reply:
Updated all view to follow wireframe! Thanks so much for the review!